nft/nft.model.js

```diff
@@ -0,0 +1,14 @@
+var mongoose = require("mongoose");
+var Schema = mongoose.Schema;
+
+var nftSchema = new Schema({
+  id: Number,
+  name: String,
+  image: String,
+  wallet_address: String,
+  likes: Number
+});
```

Comment:
You also want to have a field here to track how many times this NFT was shown to a user. The `likes` will keep the number of times the user ended up liking this NFT vs. the other.